docker/Dockerfile

```diff
@@ -7,16 +7,20 @@ ENV LANG=C.UTF-8 \
     LC_ALL=C.UTF-8 \
     HOME=/home/work
 
+ENV DEBIAN_FRONTEND=noninteractive
+
 RUN apt-get update -y && apt-get install -y build-essential libssl-dev \
     libffi-dev python3-dev libsasl2-dev libldap2-dev \
-    libsasl2-modules
+    libsasl2-modules libsasl2-2 libsasl2-modules-gssapi-mit krb5-user
     
-RUN mkdir -p $HOME/incubator-superset
+RUN mkdir -p /etc/security/keytabs/
 
 WORKDIR $HOME/incubator-superset
 
 COPY ./ ./
 
+RUN mkdir -p $HOME/incubator-superset
```

Comment:
We need to mkdir before we make it WORKDIR

Reply:
done